src/common/mmo.h

```diff
@@ -1090,6 +1090,15 @@ enum e_class {
 #endif
 };
 
+#define job_is_dummy(job) \
```

Comment:
probably this better implement as function.
Because some one may want change this list from plugin.